ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -40,8 +66,34 @@ function CollectionsTable({ collections, getSortParams, hasWriteAccess }: Collec
         });
     }
 
+    const searchValue = Array.isArray(searchFilter.Collection)
```

Comment:
An array value shouldn't happen in normal circumstances via the UI, but is here in case of a future logic change or someone manually editing URL search parameters.

Reply:
As a delayed reaction, can you write a comment, so future readers understand the reason?